### ISSUE TITLE
HSTNS-PL18 compatibility

### ISSUE THREAD

**Issue by nikosp11** [darwinbeing/HPServerPSUHack#13]
Could a similar approach be applied for a HSTNS-PL18?

**Comment by darwinbeing** [comment]
Yes, it is possible that a similar approach to modifying the HSTNS-PL18 could work as with the HSTNS-PL11. However, it would be best if you could upload some images of the power supply, including close-ups of the control board or any labels, so we can analyze the specifics.
